src/main/java/vg/civcraft/mc/civchat2/CivChat2Manager.java

```diff
@@ -59,6 +61,8 @@
 
 	private static Map<UUID, String> customNames = new HashMap<>();
 
+	private static int slowmodeTime;
```

Comment:
Good habit to include a comment explaining what unit this is in. For an integer I'd assume ticks, but that's actually not the case here. Generally use either ticks if it's an int or milliseconds (stored in a long) if its actual time. There's a method in ConfigParsing.parseTime in CMC which does basically everything you need in that regard

Reply:
Also this has no reason to be static

Reply:
Is this even set from anywhere or just left to a global default? Definitely should be loaded from the config

Reply:
Its set with the /globalslowmode set <seconds> command. I made this so superfriends can make arguments less appealing in "!" chat.